src/utils/cio/subscriber-attributes.ts

```diff
@@ -0,0 +1,3 @@
+const INTERESTED_IN_LIFETIME_SINCE = 'interested_in_lifetime_since'
+
+export {INTERESTED_IN_LIFETIME_SINCE}
```

Comment:
we didn't want to have special strings for CIO attributes in the file we were working with so created this util file to organize any attributes we are setting in egghead-next